src/app/models/carousel-responsive-options.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface CarouselResponsiveOptions {
+  breakpoint: string;
```

Comment:
Delete my interface after the merge. I created a similar one in caoursel.component.ts